js/views/playingScreen.js

```diff
@@ -258,42 +244,46 @@ class PlayingScreen extends React.Component {
 
   /**
    * call handleVrMouseUp when mouseup was called on document
-   * @param {Event} e - event object
+   * @param {Event} event - event object
    */
-  handleVrMouseUp(e) {
-    this.props.handleVrPlayerMouseUp(e);
+  handleVrMouseUp(event) {
+    this.props.handleVrPlayerMouseUp(event);
   }
 
+  /**
+   * call handlePlayerClicked when an user clicked on document
+   * @param {Event} event - event object
+   */
   handlePlayerClicked(event) {
     if (!this.props.isVrMouseMove && !this.isMobile) {
       this.props.controller.togglePlayPause(event);
     }
     this.props.handleVrPlayerClick();
   }
 
+  /**
+   * call handlePlayerFocus when the player is in focus
+   */
   handlePlayerFocus() {
     this.props.handleVrPlayerFocus();
   }
 
-  unmuteClick(event) {
-    this.props.controller.handleMuteClick();
-  }
-
   /**
    *
    * @param {number} vrDuration - key for duraction in config
-   * @param {number} defaultDuration - default value for duration
+   * @param {number} userDefaultDuration - default value for duration
    * @returns {object} empty object or object with animationDuration
    */
-  setAnimationDuration(vrDuration, defaultDuration) {
-    var style = {};
-    defaultDuration = Utils.ensureNumber(defaultDuration, 3);
+  setAnimationDuration(vrDuration, userDefaultDuration) {
+    let style = {};
+    const functionDefaultfDuration = 3; // default value for Duration if userDefaultDuration is undefined
+    const defaultDuration = Utils.ensureNumber(userDefaultDuration, functionDefaultfDuration);
     if (
       this.props.controller.state.config.animationDurations !== null &&
```

Comment:
Hope `const userDefined = this.props.controller.state.config.animationDurations` would make your code here much easier to read.

Reply:
\+